### PR TITLE
Add test for compressed Helmholtz error

### PR DESCRIPTION
**Pull request by isuruf**
Checks that the error is of the same order as the truncation error.

### CODE REVIEW

File: test/test_kernels.py [hunk 1]
```diff
@@ -866,6 +866,81 @@ def test_m2l_toeplitz():
...
+    from pytools.convergence import EOCRecorder
+    eoc_rec = EOCRecorder()
+    h_values = 2.0**np.arange(-7, -3)
+    for h in h_values:
+        sources = (h*(-0.5+sources_grid.astype(np.float64)) + mpole_center)
+        second_center = mpole_center + h/2
+        furthest_source = np.max(np.abs(sources - mpole_center))
+        m2m_vals = [0, 0]
+        for i, (mpole_expn_class, local_expn_class) in \
+                enumerate(zip(mpole_expn_classes, local_expn_classes)):
```
Comment (inducer) [inline test/test_kernels.py:910]: Let pytest parametrize this?

Comment (isuruf) [reply test/test_kernels.py:910]: We are calculating the error between compressed vs full with full regarded as the reference value. So we need both to calculate the error.

Comment (inducer) [reply test/test_kernels.py:910]: Whoops. Good point.

**Comment by inducer** [comment]
Thanks!

**Pull request merged**